CONTRIBUTING.md

```diff
@@ -23,7 +23,7 @@ pypi. Update your installation as follows.
 
 Then, install the following python packages:
 
-    $ pip install flake8 pytest pytest-cov
+    $ pip install flake8 pytest pytest-cov coverage
```

Comment:
why do we need `coverage` ?

Reply:
because pytest-cov relies on it and we need pytest-cov for our coverage reports